builtin/providers/google/resource_monitoring_group_test.go

```diff
@@ -0,0 +1,118 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"google.golang.org/api/monitoring/v3"
+)
+
+func TestAccMonitoringGroup_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckMonitoringGroupDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccMonitoringGroup_basic,
+				Check: resource.ComposeTestCheckFunc(
+					testAccMonitoringGroupExists(
+						"google_monitoring_group.foobar"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccMonitoringGroup_update(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckMonitoringGroupDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccMonitoringGroup_basic,
+				Check: resource.ComposeTestCheckFunc(
+					testAccMonitoringGroupExists(
+						"google_monitoring_group.foobar"),
+				),
+			},
+			resource.TestStep{
+				Config: testAccMonitoringGroup_update,
+				Check: resource.ComposeTestCheckFunc(
+					testAccMonitoringGroupExists(
+						"google_monitoring_group.foobar"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccMonitoringGroupCreate(t *testing.T) {
```

Comment:
Generally we like to use the `TestAcc{ResourceName}_{variation}` format when writing unit tests. A consistent naming schema (like the ones you have above, which are great!) helps us use regular expressions to run just the tests we need, because our integration test suite is so extensive.